test/ssh.js

```diff
@@ -2,11 +2,16 @@
 
 var SSH = require('../lib/ssh.js'),
     expect = require('expect.js'),
+    fs = require('fs'),
 
     config = require('../config/ssh.json'),
     ssh;
 
 describe('SSH', function() {
+    if (config.key) {
+        config.key = fs.readFileSync(config.key);
```

Comment:
`ssh2` needs a buffer or a string with the contents of the key file (see [privateKey](https://github.com/mscdex/ssh2#client-methods)).